address feedback from allen baum on mailing list dated oct 29, 2024 (#110)
riscv/riscv-smmtt: chapter3.adoc

`MFENCE.SPA` is only valid in M-mode. If operand rs1≠x0, it
specifies a single physical address, and if rs2≠x0, it specifies a single SDID.

> If the contents of rs1 corresponds to an `MPT_L1_DIR` mapping, the fence applies to
> a 4 KiB page. If the contents of rs1 corresponds to a `2M_PAGES` or `4M_PAGES` mapping,
> the fence applies to a 2 MiB or 4 MiB range. If the contents of rs1 corresponds to one

Reviewer: maybe a bit clearer:

```suggestion
If the address in rs1 corresponds to an `MPT_L1_DIR` mapping, the fence applies to
a 4 KiB page. If the address in rs1 corresponds to a `2M_PAGES` or `4M_PAGES` mapping,
the fence applies to a 2 MiB or 4 MiB range. If the address in rs1 corresponds to one
```
